lib/binaries/win-utils.ts

```diff
@@ -104,6 +104,6 @@ export async function copyNeededDlls(
     const dlls = await winutils.get_dlls_used(executableFilename);
     for (const dll of dlls) {
         const infolder = path.join(dirPath, path.basename(dll));
-        await fs.copy(dll, infolder);
```

Comment:
@partouf fyi. I'm 99.9999% sure this is equivalent in the cases we're using (copying _files_ not directories) but wanted to call it out.